### PR TITLE
Update sdl scripts and templates and re-enable sdl runs

### PR DESCRIPTION
**Pull request by riarenas**
https://github.com/dotnet/arcade/issues/8265

Tested this here: https://dev.azure.com/dnceng/internal/_build/results?buildId=1539902&view=results

We also got a false positive for credscan so that's something?

### CODE REVIEW

**Review by MattGal — APPROVED** [review]
LGTM, you could consider adding  exceptions files at the same time.

**Comment by riarenas** [comment]
Yep I'll test out the suppressions and add them.

**Comment by riarenas** [comment]
Was able to get the policheck exclusions working. Still trying to figure out how to get the credscan suppression https://dev.azure.com/dnceng/internal/_build/results?buildId=1540138&view=results

**Comment by riarenas** [comment]
Credscan suppressions working now as well: https://dev.azure.com/dnceng/internal/_build/results?buildId=1541839&view=results

**Comment by riarenas** [comment]
Also sneaking in a change to use windows-latest for the approval stage to  prepare for the win-2016 environment removal.

File: CredScanSuppressions.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+      {
+        "placeholder": "  hasWebHookSecret: boolean",
+        "_justification": "The word secret is not a secret"
+      }
```
Comment (MattGal) [inline CredScanSuppressions.json:7]: Deep insight.

**Review by MattGal — APPROVED** [review]
LGTM, awesome to see a clean SDL run

File: CredScanSuppressions.json (hunk 1 above)
Comment (riarenas) [reply CredScanSuppressions.json:7]: If only we could teach that to the credscan ML

**Comment by riarenas** [comment]
Oh wow I checked in a commit that was for test only that removes a bunch of build legs. Need to fix that asap

**Comment by riarenas** [comment]
https://github.com/dotnet/arcade-services/pull/1796 fixes it

**Pull request merged**